libs/constrained-generators/src/Constrained/Examples/Basic.hs

```diff
@@ -312,3 +312,14 @@ whenTrueExists = constrained $ \x ->
       [ not_ [var| b |]
       , not_ (not_ b)
       ]
+
+wtfSpec :: Specification BaseFn ([Int], Maybe ((), [Int]))
+wtfSpec = constrained' $ \ [var| options |] [var| mpair |] ->
```

Comment:
@MaximilianAlgehed What kind of naming is this? Could you please submit a PR that fixes it to something a bit more professional.